.github/workflows/etcd-defrag.yaml

```diff
@@ -0,0 +1,30 @@
+name: Publish
+
+on:
+- push
+- workflow_dispatch
+
+jobs:
+  goversion:
+    uses: ./.github/workflows/go-version.yaml
+  publish-tagged:
+    if: |
+      startsWith(github.ref, 'refs/tags/v') &&
+      !github.event.pull_request.head.repo.fork
+    runs-on: ubuntu-latest
+    needs: goversion
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ${{ needs.goversion.outputs.goversion }}
+      - uses: ko-build/setup-ko@v0.6
+        name: Setup ko
+      - name: Run ko publish for ghcr.io
+        env:
+          KO_DOCKER_REPO: ghcr.io/${{ github.repository_owner }}
+        run: |
+          export COMMIT=${{ github.sha}}
+          export TAG=$(echo ${{ github.ref }} | cut -d "/" -f 3 - )
+          ko publish ./ --base-import-paths --platform=linux/amd64,linux/arm64 --tags $TAG
+          ko publish ./ --base-import-paths --platform=linux/amd64,linux/arm64
```

Comment:
The line 29 has flag `--tags $TAG`, but line 30 doesn't have. Does it mean that line 30 generate an image with tag "`latest`"?

Reply:
exactly

Reply:
thx